cpp/src/io/csv/csv_reader.cu

```diff
@@ -227,8 +227,10 @@ gdf_error read_csv(csv_read_arg *args)
 
 	if (args->delim_whitespace) {
```

Comment:
They are not supposed to be the same. if delimiter is ' ', it will match only single spaces. delim_whitespace implies that is will match continuous spaces as a single delimiter.